Services/TMS/TrainingSearch/classes/BookableCourse.php

```diff
@@ -221,8 +228,7 @@ public function getImportantFields() {
 
 	public function getFurtherFields() {
 		// Take info 2 to end as fields in header line
-		$short_info = $this->getShortInfo();
-		return $this->unpackLabelAndValue(array_slice($short_info, 2));
```

Comment:
Hallo Stefan, ich hatte das hier so gebaut, dass er alle außer die ersten zwei Felder in die Seitenleiste passt. Da die Infos in der Zeile verschwinden, wenn das Ding ausgeklappt wird, würde ich die einfach oben als erstes in der Seitenleiste weiter anzeigen. Ich merge das aber nun trotzdem und mache die entsprechenden Umbauten. Danke!